12.glTF/main.cpp

```diff
@@ -351,8 +356,9 @@ class GLTFApp : public ApplicationBase
 				// first uint needs to be the count
 				allNodes.insert(allNodes.begin(),allNodes.size());
 				pivotNodesRange.offset += sizeof(uint32_t);
-
-				auto allNodesBuffer = utilities->createFilledDeviceLocalBufferOnDedMem(transferUpQueue,sizeof(scene::ITransformTree::node_t)*allNodes.size(),allNodes.data());
+				video::IGPUBuffer::SCreationParams params;
+				params.size = sizeof(scene::ITransformTree::node_t) * allNodes.size();
```

Comment:
again, params needs `usage`, https://github.com/Devsh-Graphics-Programming/Nabla-Examples-and-Tests/pull/3/files#r912920764
That's why I insist testing on Vulkan which I'm pretty sure you didn't test on, sorry for not mentioning it earlier